src/test/java/fi/aalto/cs/intellij/common/BuildInfoTest.java

```diff
@@ -0,0 +1,114 @@
+package fi.aalto.cs.intellij.common;
+
+import java.io.ByteArrayInputStream;
+import java.util.Properties;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BuildInfoTest {
+
+  @Test
+  public void testCreateVersion() {
+    String versionString = "3.5.24";
+    BuildInfo.Version version = new BuildInfo.Version(3,5,24);
+
+    Assert.assertEquals(3, version.major);
```

Comment:
I'd suggest to import static for the cases when it does not reduce readability

Reply:
A good point! Much better readability now. 👍